documentation/Functions.md

```diff
@@ -322,16 +322,27 @@ downloadFile("https://bit.ly/2BCkFa9", "file", "downloads")
 ***
 
 ### buildNg <a name="buildNg"></a>
+This function is used to build angular project.
```

Comment:
This function is used to create an Angular project.

Reply:
changed